models/face_detection_yunet/README.md

```diff
@@ -7,6 +7,15 @@ Notes:
 - For details on training this model, please visit https://github.com/ShiqiYu/libfacedetection.train.
 - This ONNX model has fixed input shape, but OpenCV DNN infers on the exact shape of input image. See https://github.com/opencv/opencv_zoo/issues/44 for more information.
 
+Results of accuracy evaluation with [tools/eval](../../tools/eval).
+
+| Models      | Easy AP | Medium AP | Hard AP | 
+|-------------|---------|-----------|---------|
+| YuNet       | 0.8498  | 0.8384    | 0.7357  |
+| YuNet quant | 0.7751  | 0.8145    | 0.7312  |
```

Comment:
Medium AP is much higher than Easy AP for YuNet quant?

Reply:
@WanliZhong You can doublecheck with the offcial eval tools of WiderFace

Reply:
The result of the original script is also the same result.
![image](https://user-images.githubusercontent.com/49567307/178895008-e4fcb790-2832-4894-8d32-1cdc75c1821e.png)